package.json

```diff
@@ -23,7 +23,7 @@
   ],
   "scripts": {
     "test": "jest",
-    "typescript": "tsc --noEmit",
+    "typescript": "tsc",
```

Comment:
can you please explain the problem with `--noEmit`?

Reply:
I think this was an artifact of my trying to figure out the build process